src/components/application_manager/include/application_manager/application_impl.h

```diff
@@ -341,6 +341,23 @@ class ApplicationImpl : public virtual Application,
   AppExtensionPtr QueryInterface(AppExtensionUID uid) OVERRIDE;
 #endif
 
+  /**
+   * @breif Allows to save mobile's command smart object in order
```

Comment:
@LitvinenkoIra I would like to avoid duplication description it is hard to support.  Only in case if derived class description contains some important notes that differ from base interface description